get_bullshtml.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -uex
+
+bullshtml_vers='master'
+bullshtml_url='https://github.com/simplivity/bullshtml/archive/'
+bullshtml_dnl="${bullshtml_vers}.tar.gz"
+bullshtml_tar="bullshtml-${bullshtml_vers}.tar.gz"
+
+if [ -e "${bullshtml_tar}" ]; then
+  zflag=" -z ${bullshtml_tar}"
+else
+  zflag=
+fi
+
+curl --location --retry 10 --retry-max-time 60 --silent --show-error \
+    ${zflag} "${bullshtml_url}${bullshtml_dnl}" -o "${bullshtml_tar}"
+
+if [ "${bullshtml_tar}" -nt bullshtml ]; then
+  rm -rf bullshtml
+  mkdir -p bullshtml
+  tar -C bullshtml --strip-components=1 -xf "${bullshtml_tar}"
+  pushd bullshtml
+    if [ ! -e build ]; then
+      ./gradlew build
```

Comment:
Builder prerequisite?  Perhaps a comment here about where this comes from.  How to get it installed.  Which RPM one needs to install.

Would it just be easier to do all of this in a docker container, to maintain our stance that builders simply need to be able to run docker, and not encumber them with having to include a growing list of prerequisites.